app/controllers/oligrapher_controller.rb

```diff
@@ -94,6 +94,27 @@ def find_nodes
     render json: entities
   end
 
+  def find_connections
```

Comment:
This route could use a request spec which would go here: https://github.com/public-accountability/littlesis-rails/blob/master/spec/requests/entities_spec.rb

Reply:
Ah yes, but I'll put it in the oligrapher request spec: https://github.com/public-accountability/littlesis-rails/blob/master/spec/requests/oligrapher_request_spec.rb